src/background.js

```diff
@@ -1419,9 +1419,12 @@ function initializePopup() {
  * @memberOf Background
  */
 function addCommonGhosteryAndAntitrackingListeners() {
-	chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, {
-		urls: ['http://*/*', 'https://*/*']
-	}, ['blocking']);
+	// Edge doesn't support WebSockets
+	const urls = ['http://*/*', 'https://*/*'];
+	if (!IS_EDGE) {
+		urls.push('ws://*/*', 'wss://*/*');
+	}
+	chrome.webRequest.onBeforeRequest.addListener(onBeforeRequest, { urls }, ['blocking']);
```

Comment:
@christophertino is there a reason why we don't also add these to the `onHeadersReceived` listener?